src/Configuration/Configuration.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration\Configuration;
+
+use Illuminate\Support\Arr;
+use MyParcelCom\Integration\Configuration\Properties\PropertyCollection;
+use MyParcelCom\Integration\ProvidesJsonAPI;
+
+class Configuration implements ProvidesJsonAPI
+{
+    /**
+     * @param PropertyCollection $properties
+     */
+    public function __construct(private readonly PropertyCollection $properties)
+    {
+    }
+
+    public function transformToJsonApiArray(): array
+    {
+        return [
+            '$schema'              => 'http://json-schema.org/draft-04/schema#',
```

Comment:
Lets nest in something like `configuration_schema`  field so we can have a way to add more information if needed.